examples/testapp/next.config.mjs

```diff
@@ -7,4 +9,21 @@ export default {
     // GitHub discussion for supporting biome: https://github.com/vercel/next.js/discussions/59347
     ignoreDuringBuilds: true,
   },
+  webpack: (config, { isServer }) => {
+    // Exclude Node.js built-in modules from client bundle
+    if (!isServer) {
+      config.resolve.fallback = {
+        ...config.resolve.fallback,
+        zlib: false,
+      };
+
+      // Ignore node:zlib imports in client-side code
+      config.plugins.push(
+        new webpack.IgnorePlugin({
+          resourceRegExp: /^node:zlib$/,
+        })
+      );
+    }
+    return config;
+  },
```

Comment:
assuming consumers of our SDK don't have to do this?

Reply:
Updated to Implement environment specific exports (and removed this)